packages/describe-package/src/api.ts

```diff
@@ -43,23 +44,29 @@ export type ParameterDescription = {
   type: string; // this is a human-readble string I think. Should we also store a machine readable type?
 };
 
-// I think we can get the API down to just these functions
-
 export const describePackage = async (
   specifier: string,
   _options: Options
 ): Promise<PackageDescription> => {
   const { name, version } = getNameAndVersion(specifier);
   const project = new Project();
 
-  const files = await fetchDTSListing(specifier);
+  // Include language-common in the project model
+  // (I don't expect this to be permanent)
+  const common = await fetchDTSListing('@openfn/language-common');
+  for await (const fileName of common) {
+    const f = await fetchFile(`@openfn/language-common${fileName}`);
+    // Flatten the paths or else there's trouble
+    // TODO need to better understand this at some stage
+    const relativeFileName = fileName.split('/').pop();
```

Comment:
This is a bit janky - but common's paths are nested in `types`, so you have paths like `/types/Adaptor.d.ts` coming out of the package.json.

This basically breaks the tracking in the TS project. Flattening the paths to simple file names seems to work better.

I would like a better solution but I'll sort it out when we do proper dynamic dts dependency tracking.